hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/analysis/HoodieAnalysis.scala

```diff
@@ -350,6 +353,35 @@ case class HoodieResolveReferences(sparkSession: SparkSession) extends Rule[Logi
         l
       }
 
+    case TimeTravelRelation(plan: UnresolvedRelation, timestamp, version) =>
+      // TODO: How to use version to perform time travel?
```

Comment:
> do you have a follow-up ticket for supporting version-only time-travel?

I have no plan for this one at present.

Reply:
> > do you have a follow-up ticket for supporting version-only time-travel?
> 
> I have no plan for this one at present. I'll look at the implementation based on version time travel later

@XuQianJin-Stars since we just need to implement instant as of, then there's no strong requirement to support `version`

Reply:
> > > do you have a follow-up ticket for supporting version-only time-travel?
> > 
> > 
> > I have no plan for this one at present. I'll look at the implementation based on version time travel later
> 
> @XuQianJin-Stars since we just need to implement instant as of, then there's no strong requirement to support `version`

well